src/mbgl/annotation/annotation_manager.cpp

```diff
@@ -1,7 +1,7 @@
 #include <mbgl/annotation/annotation_manager.hpp>
 #include <mbgl/annotation/annotation_tile.hpp>
-#include <mbgl/style/style.hpp>
 #include <mbgl/layer/symbol_layer.hpp>
+#include <mbgl/style/style.hpp>
```

Comment:
They're typically roughly in order of "importance", as best I can read. So in this case, we've got our own class header, then the thing we produce, then the thing we use to produce that, then a specific class of many that we need access to. At least that's my logic flow in looking at the old version. Maybe we should settle on a stricter system.